test/WebJobs.Script.Tests.Integration/WebJobs.Script.Tests.Integration.csproj

```diff
@@ -50,6 +50,7 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="6.0.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.4" />
     <PackageReference Include="Moq" Version="4.9.0" />
+    <PackageReference Include="System.Reactive.Core" Version="5.0.0" />
```

Comment:
Even after the above fixture fixups, I was seeing an assembly load failure for System.Reactive.Core. Need to figure out why references to the Proxies package aren't adding this required assembly.